system/lib/wasmfs/file.h

```diff
@@ -147,4 +133,15 @@ class Directory : public File {
   Handle locked() { return Handle(shared_from_this()); }
 };
 
+class MemoryFile : public DataFile {
```

Comment:
Please add a comment explaining what this file type is.

Reply:
I made a short comment, but we can add some additional comments later.